.github/workflows/nix.yaml

```diff
@@ -14,14 +14,14 @@ permissions:
 jobs:
   tenzir:
     name: Tenzir
```

Comment:
```suggestion
    name: Tenzir (${{ runner.os }} ${{ runner.arch }})
```

Reply:
Unfortunately the workflow fails with this change applied. It seems the runner context is not available in this scope.

Reply:
Just leave it out then.